### DISCUSSION TITLE
O que você achar de um livro introdutório do MEAN stack em português?

### DISCUSSION THREAD

**Discussion by JorgeGuerraPires** [frontendbr/forum#1810] [Discussão]
Olá! 

Estou com um projeto para o futuro, de escrever um livro sobre mean stack, resultado de mais de uma no de trabalho. Você acha que valeria a pena escrever em português?

**Comment by ninetails** [comment]
Há diversas formas, eu sou bem leigo no assunto e não cheguei a escrever livro algum. Porém jogo algumas idéias:

A Casa do Código já tem um livro sobre [MEAN Stack](https://www.casadocodigo.com.br/products/livro-mean?_pos=1&_sid=3c29ff6d0&_ss=r), porém ao mesmo tempo também tem essa seção [quero ser um autor](https://www.casadocodigo.com.br/pages/quero-ser-um-autor). Diria talvez pra comprar o livro já lançado, conversar com o autor e fazer um outro abordando o que não foi visto no primeiro, complementando-o.

Já vendo alternativas, não conheço muito bem como é esse mercado editorial. Talvez tentar ir atrás de livros publicados em PTBR e conversar com seus autores. (Até devem ter alguns aqui no fórum)

Uma última proposta, falaria pra ver de publicar via [Leanpub](https://leanpub.com/).

Sobre edição, se quiser algo open (no final a pessoa pode optar por ler íntegro online de graça ou se quiser o físico, comprar) no próprio GitHub ou ver plataformas como [GitBook](https://www.gitbook.com/pricing).

Já sobre valer a pena, esperaria ver galera comentando aqui ou vc mesmo pode ir perguntando em eventos etc. Ou se quiser mesmo levar a sério, fazer um estudo de caso envolvendo pesquisar com devs se há a necessidade de ter um livro sobre esse assunto e se os livros e cursos atuais não suprem essa demanda. Ou por último ponto, somente faça se estiver com vontade, como se fosse um _hobby_. Talvez para este último, se vc for fazer algo mais voltado pra open....(sei lá, open knowledge pq não é bem open source rs) faça-o em artigos e no final compile num livro. Um exemplo bom é o livro do [Eric Elliot, _Composing Software_](https://medium.com/javascript-scene/composing-software-the-book-f31c77fc3ddc). O lado bom deste último é que vc já vai tendo _feedbacks_ enquanto escreve, sendo que fazendo "fechado" vc só vai conseguir _feedbacks_ de amigos a quem vc compartilhar os _beta versions_.

**Comment by JorgeGuerraPires** [comment]
Carlos Kazuo, muito obrigado pela reposta detalhada! Muito bom seus insights! Vou pensar um pouco sobre o que coloca.  Estava pensando em usar Amazon, parece fácil publicar com eles, e pensava no que falou, deixar o livro open, e se a pessoa quiser, compra como forma de ajudar o autor.

**Comment by JorgeGuerraPires** [comment]
Estou acabando um postdoc, onde foquei em MEAN stack, estava querendo fechar esse período com algo além das formalidades, como relatórios finais, o livro seria essa vontade na prática.
